test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -506,7 +506,7 @@ protected bool CheckEqualityFromPathsCore(string relPath, string basePath, strin
 
         private void GetNumbersFromFile(ref string firstString, ref string secondString, int digitsOfPrecision)
         {
-            Regex _matchNumer = new Regex(@"-?[0-9]+\.?[0-9]*(E-[0-9]*)?\b", RegexOptions.IgnoreCase | RegexOptions.Compiled);
+            Regex _matchNumer = new Regex(@"-?\b[0-9]+\.?[0-9]*(E-[0-9]*)?\b", RegexOptions.IgnoreCase | RegexOptions.Compiled);
```

Comment:
The `\b` needs to go before the negative sign, right?

Reply:
no its not , the way ```\b``` works it doesnot consider ```-`` character as start of word boundary. so ```\b-?``` is actually same as ```\b``` which wont catch the ```-```.

Reply:
OK, I'm fine with this then. It would make a good comment explaining exactly what this regex is trying to match.